### PR TITLE
[WIP] Homebrew: Allow install from HEAD

### PR DESCRIPTION
**Pull request by TangRufus**
https://github.com/TangRufus/homebrew-tap/blob/feac3f9776e86d7e65910b7a5cafd00e30b56e76/Formula/trellis-cli.rb works, but not sure how to test goreleaser generates the formula as expected.

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -38,8 +38,16 @@ brews:
     folder: Formula
     homepage:  https://roots.io/trellis
     description: A CLI to manage Trellis projects
+    custom_block: |
+      head "https://github.com/roots/trellis-cli.git"
+      depends_on "go" => :build
```
Comment (swalkinshaw) [inline .goreleaser.yml]: As long as this means it's only needed for build/head 👍

Comment (TangRufus) [reply .goreleaser.yml]: Good catch!
Seems a goreleaser tap always install build dependencyies.

Comment (TangRufus) [reply .goreleaser.yml]: Seems goreleaser always install `depends_on "go" => :build`.

What do you think if we make another formula (trellis-cli-dev) in https://github.com/roots/homebrew-tap

Something like what sequel does:
```sh-session
$ brew search sequel
==> Casks
psequel                    sequel-pro                 sequel-pro-nightly ✔
```

Comment (swalkinshaw) [reply .goreleaser.yml]: I'm good with that 👍 I think that just means adding another brew entry in the goreleaser config in this repo?

Comment (TangRufus) [reply .goreleaser.yml]: I think https://github.com/roots/homebrew-tap/pull/1 is enough because that formula doesn't need to be changed by goreleaser

**Comment by swalkinshaw** [comment]
`brew install roots/trellis-cli-dev` should work now

**Pull request closed without merging**